test/tests/compute/test_rackhd20_computenode_pollers.py

```diff
@@ -23,26 +23,6 @@
     sys.exit(255)
 
 
-def get_rackhd_nodetype(nodeid):
-    nodetype = ""
-    # get the node info
-    mondata = fit_common.rackhdapi("/api/2.0/nodes/" + nodeid)
-    if mondata['status'] != 200:
-        print "Incorrect HTTP return code on nodeid, expected 200, received: {}".format(mondata['status'])
-    else:
-        # get the sku id contained in the node
-        sku = mondata['json'].get("sku")
-        sku = sku.split("/")[-1]
-        if sku:
-            skudata = fit_common.rackhdapi("/api/2.0/skus/" + sku)
-            if skudata['status'] != 200:
-                print "Error: Incorrect HTTP return code on sku, expected 200, received: {}".format(skudata['status'])
-            else:
-                nodetype = skudata['json'].get("name")
-        else:
-            print "Error: nodeid {} did not return a valid sku in get_rackhd_nodetype{}".format(nodeid, sku)
-    return nodetype
-
 from nose.plugins.attrib import attr
```

Comment:
sounds good. need to also clean up the calls in this script that are outside of the class, so will do both.